### PR TITLE
Add background

### PR DESCRIPTION
**Pull request by LeandroStanger**
The Background is used in the App Center of Endless OS
An example image
Captura de tela de 2019-05-11 20-02-07

### CODE REVIEW

File: com.valvesoftware.Steam.appdata.xml [hunk 1]
```diff
@@ -38,6 +38,9 @@
       <image type="source" width="1280" height="720">https://raw.githubusercontent.com/flathub/com.valvesoftware.Steam/master/com.valvesoftware.Steam.screenshot-2.png</image>
     </screenshot>
   </screenshots>
+  <metadata>
+    <value key="GnomeSoftware::popular-background">https://raw.githubusercontent.com/flathub/com.valvesoftware.Steam/master/com.valvesoftware.Steam-thumb.png</value>
```
Comment (gasinvein) [inline com.valvesoftware.Steam.appdata.xml]: Where does this picture come from? Are we allowed to use it?

Comment (LeandroStanger) [reply com.valvesoftware.Steam.appdata.xml]: Pixabay License
Free for commercial use
No attribution required
https://pixabay.com/photos/game-remote-gamer-video-2294201

**Comment by LeandroStanger** [comment]
bot, build com.valvesoftware.Steam

**Comment by nanonyme** [comment]
Will users not get confused by that default picture is in Spanish?

**Comment by LeandroStanger** [comment]
@nanonyme Spanish?

**Comment by nanonyme** [comment]
Sorry for the confusion, Portuguese.

**Comment by nanonyme** [comment]
I think ideally if we want to be language-neutral the image should have no text.

**Comment by LeandroStanger** [comment]
com valvesoftware Steam-thumb
The image has no text

**Comment by gasinvein** [comment]
Does this image have any text on in it? I've failed to see.
Anyway, wouldn't a picture of Steam Controller (instead of Xbox One pad) suit Steam better?

**Comment by LeandroStanger** [comment]
I did not find a Steam Controller image
https://pixabay.com/images/search/steam%20controller/

**Comment by nanonyme** [comment]
Ah, sorry. I got confused by that initial example. This looks fine.

**Comment by nanonyme** [comment]
That releases thing might make things look a bit weird since it's just the version of the bootstrapper but I guess it's fine.

**Pull request merged**